src/components/TrackFilePicker.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from "prop-types";
+import Select from "react-select";
+
+
+
+//handleInputChange takes in a string trackType
+export const TrackFilePicker = ({
+  tracks, // array of available tracks
```

Comment:
What format should this array be in? Can you refer back to the types file? Because the demo example value doesn't look that much like a real set of available tracks.